functions/process.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+
+def replace_ensembl_ids(expression_df,
+gene_id_mapping):
```

Comment:
`gene_id_mapping should either be indented to be level with `expression_df` or the newline between them should be removed